src/database/migrations/2020_09_19_170010_create_corona_status_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+class CreateCoronaStatusTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('corona_status', function (Blueprint $table) {
+            $table->id();
+            $table->string('code')
+                ->unique();
+            $table->boolean('is_affected')
+                ->index('is_affected');
+            $table->timestamp('created_at')
+                ->default(DB::raw('CURRENT_TIMESTAMP'));
+            $table->timestamp('updated_at')
+                ->default(DB::raw('CURRENT_TIMESTAMP on update CURRENT_TIMESTAMP'));
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
```

Comment:
```suggestion
    public function down(): void
```
Šalink `php doc` ir pridėk grąžinimo tipą.